spyder/config/base.py

```diff
@@ -39,9 +39,14 @@
 TEST = os.environ.get('SPYDER_TEST')
 
 
-# To do some adjustments for pytest
-# This env var is defined in runtests.py
-PYTEST = os.environ.get('SPYDER_PYTEST')
+def running_under_pytest():
+    """
+    Return True if currently running under py.test.

```

Comment:
This is trivially minor, but though while PEP 257 allows it, it seems to be convention that we put the summary one-liner on the first line, next to the ``'''``.

Reply:
Really? I always thought it was the other way around ... I had a quick and random look in our code base. To me, it looks like we have far too few multi-line docstrings and that there is no clear preference for either style. We may need an executive decision here.

Reply:
Hmmm actually I like either


```
def func():
    """Summary."""
```

or

```
def func():
    """
    Summary.

    Something extra.
    """
```

I don't like this at all :-p

```
def func():
    """Summary.

    Something extra.
    """
```

Reply:
+1 for

```python
def func():
    """
    Summary.

    Some
    """
```

I like to write docstrings this way too.